build-caching-maven-samples/enforcer-project/pom.xml

```diff
@@ -128,118 +137,31 @@
-                          <name>rules</name>
-                          <inputs>
-                            <properties>
-                              <property>
-                                <name>includes</name>
-                              </property>
-                              <property>
-                                <name>excludes</name>
-                              </property>
-                              <property>
-                                <name>searchTransitive</name>
-                              </property>
-                              <property>
-                                <name>message</name>
-                              </property>
-                            </properties>
-                          </inputs>
-                        </property>
-                      </iteratedProperties>
-                      <nestedProperties>
-                        <property>
-                          <name>project</name>
-                          <iteratedProperties>
-                            <property>
-                              <name>dependencies</name>
-                              <inputs>
-                                <properties>
-                                  <property>
-                                    <name>groupId</name>
-                                  </property>
-                                  <property>
-                                    <name>artifactId</name>
-                                  </property>
-                                  <property>
-                                    <name>version</name>
-                                  </property>
-                                  <property>
-                                    <name>type</name>
-                                  </property>
-                                  <property>
-                                    <name>scope</name>
-                                  </property>
-                                </properties>
-                              </inputs>
-                            </property>
-                          </iteratedProperties>
-                        </property>
-                      </nestedProperties>
                       <outputs>
                         <cacheableBecause>banned dependency check should run when dependencies change</cacheableBecause>
                       </outputs>
                     </execution>
                     <execution>
                       <id>enforce-bytecode-version</id>
                       <inputs>
+                        <properties>
+                          <property>
+                            <name>rules</name>
+                          </property>
+                        </properties>
                         <ignoredProperties>
-                          <ignore>commandLineRules</ignore>
+                          <ignore>rulesToSkip</ignore>
+                          <ignore>rulesToExecute</ignore>
```

Comment:
`rulesToSkip / rulesToExecute` shouldn't be ignored imo, I assume that changing one of those value would impact the goal outcome.
This being said, as it is not ignored for the first execution, that might be fine for `enforce-bytecode-version`

Reply:
> ...as it is not ignored for the first execution, that might be fine for enforce-bytecode-version

Just so I understand, why would that make it fine?

Reply:
Just a supposition as my understanding was that it was set on purpose, please ignore then